### DISCUSSION TITLE
MIS CONTRIBUCIONES NO SE MUESTRAN EN MI PERFIL

### DISCUSSION THREAD

**Discussion by josuereyes16** [community/community#125197] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by davevad93** [comment, marked as answer]
Hi @josuereyes16 ,

It could be for several reasons, take a look at [this](https://docs.github.com/es/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile), maybe it can help you to find out why.

**Comment by ebndev** [comment]
Hi @josuereyes16, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
